code/modules/clothing/spacesuits/void/merc.dm

```diff
@@ -41,6 +41,6 @@
 	desc = "An advanced suit that protects against injuries during special operations. Property of Gorlex Marauders."
```

Comment:
While you at it, can you please change "Gorlex Marauders" to something else?

Reply:
Yeah, sure, why not? It kinda stuck out to me too, but I didn't want to touch descriptions if someone else already did a pass over them.